src/vmm/src/device_manager/mmio.rs

```diff
@@ -578,19 +570,18 @@ mod tests {
         }
     }
 
+    fn default_mmio_manager() -> MMIODeviceManager {
```

Comment:
This function is same as one in `src/vmm/src/builder.1026`. Why do we even need them if they are 1 line anyway?

Reply:
The fact that we ended up with 2 `default_mmio_manager` kinda speaks for itself. `MMIODeviceManager::new(Rc::new(ResourceAllocator::new().unwrap())).unwrap()` might be ugly to look at, but it still has no "logic" inside, so I see no reason to shove it into separate "helper" method. I any case, at least let's use  function from `mmio.rs` and delete one from `bulder.rs`.

Reply:
> The fact that we ended up with 2 default_mmio_manager kinda speaks for itself

I don't understand what this means. I really don't think it's that important, but I am happy to use the other one.

Reply:
maybe we can just implement `Default` here?

Reply:
> The fact that we ended up with 2 default_mmio_manager kinda speaks for itself

You created second version of a function that already existed, which means that the original function was not known to you. If instead we would have used usual `MMIODeviceManager` constructor everywhere, this problem would have never existed. In the end you still needed to update multiple places where `MMIODeviceManager` is created, so the argument for "change in one place" is also questionable.